### PR TITLE
Correct description of transform-origin

### PR DESCRIPTION
**Pull request by brady-ds**
* **Problem:** The description of `transform-origin` should explain that, effectively, the transform origin is translated to (0, 0), the CSS `transform` property is applied so that those transformations are about the transform origin, and then the transform origin is translated back to its original location.  Instead, the description says the reverse: (0, 0) is translated to the transform origin, the `transform` property is applied, and then (0, 0) is restored back to its original location.  This would incorrectly make the transformations apply around the point mirroring the transform origin.  The code example immediately following also contradicts the wrong description.

* **URL:** <https://developer.mozilla.org/en-US/docs/Web/CSS/transform-origin>

* **Issue:** None

### CODE REVIEW

File: files/en-us/web/css/transform-origin/index.html [hunk 1]
```diff
@@ -19,8 +19,7 @@
 
 <p>The transformation origin is the point around which a transformation is applied. For example, the transformation origin of the <code><a href="/en-US/docs/Web/CSS/transform-function/rotate">rotate()</a></code> function is the center of rotation.</p>
 
-<p>This property is applied by first translating the element by the value of the property, then applying the element's transform, then translating by the negated property value.<br>
- This means, this definition</p>
+<p>This property is applied by first translating the element by the negated value of the property, then applying the element's transform, then translating by the property value. This means, this definition</p>
```
Comment (ericwbailey) [inline files/en-us/web/css/transform-origin/index.html:22]: I don't think "by the negated value of the property" is clear enough. Can we rephrase it to be more descriptive, like in your PR description?

**Review by ericwbailey — CHANGES_REQUESTED** [review]

**Comment by brady-ds** [comment]
Besides expanding the indicated paragraph as requested, I also added [b0382b2](https://github.com/mdn/content/pull/4925/commits/b0382b29c62f5470af7d52b5934071d99a68c298) to clean up language in the prior paragraph that was inconsistent with the property name and therefore inconsistent with the text I added.  Since it could be considered an unrelated correction, please let me know if you would prefer it as a separate PR.

File: files/en-us/web/css/transform-origin/index.html [hunk 2]
```diff
@@ -31,6 +30,8 @@
 <pre class="brush: css">transform-origin: 0 0;
 transform: translate(-100%, 50%) rotate(45deg) translate(100%, -50%);</pre>
 
+<p>where, reading from right to left, <code>translate(100%, -50%)</code> is the translation to bring the transform origin to the true origin, <code>rotate(45deg)</code> is the original transformation, and <code>translate(-100%, 50%)</code> is the translation to restore the transform origin to its original location.</p>
```
Comment (ericwbailey) [inline files/en-us/web/css/transform-origin/index.html:33]:
```suggestion
<p>Reading from right to left, <code>translate(100%, -50%)</code> is the translation to bring the transform origin to the true origin, <code>rotate(45deg)</code> is the original transformation, and <code>translate(-100%, 50%)</code> is the translation to restore the transform origin to its original location.</p>
```

**Review by ericwbailey — CHANGES_REQUESTED** [review]
This is great, thank you. One last little tweak, and then I think this is good to go!

**Review by ericwbailey — APPROVED** [review]
Wonderful. Thank you for your effort on this, and apologies for the delay in getting back to you. Also, congrats on your first MDN contribution!
